### PR TITLE
Update README to new structure

### PR DESCRIPTION
**Pull request by fmauch**
Updates the README according to the changes from #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +111,53 @@ void YourRobot::cancelCartesianInterpolation()
...
+  switch (result)
+  {
+    case robot_vendor::TrajectoryResult::TRAJECTORY_RESULT_SUCCESS:
+    {
+      final_state = hardware_interface::ExecutionState::SUCCESS;
+      break;
+    }
+    case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_CANCELED:
+    {
+      final_state = hardware_interface::ExecutionState::SUCCESS;
```
Comment (stefanscherzinger) [inline README.md]:
```suggestion
      final_state = hardware_interface::ExecutionState::PREEMPTED;
```

File: README.md [hunk 2]
```diff
@@ -131,6 +111,53 @@ void YourRobot::cancelCartesianInterpolation()
...
+      break;
+    }
+    case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_FAILUDE:
+    {
+      final_state = hardware_interface::ExecutionState::SUCCESS;
```
Comment (stefanscherzinger) [inline README.md]:
```suggestion
      final_state = hardware_interface::ExecutionState::ABORTED;
```

**Comment by stefanscherzinger** [comment]
Thanks @fmauch for thinking of updating this. I almost forgot. There's even more to update in the README.md. For instance, [the section on the controller.yaml](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/tree/master/pass_through_controllers#controller-yaml), and I guess [the examples no longer exist](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/tree/master/pass_through_controllers#examples) (they became integration tests)

File: README.md [hunk 3]
```diff
@@ -131,6 +111,53 @@ void YourRobot::cancelCartesianInterpolation()
...
+      break;
+    }
+    case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_CANCELED:
+    {
+      final_state = hardware_interface::ExecutionState::PREEMPTED;
+      break;
+    }
+    case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_FAILUDE:
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
    case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_FAILURE:
```

Comment (fmauch) [reply README.md]: thanks for spotting this!

**Comment by fmauch** [comment]
> Thanks @fmauch for thinking of updating this. I almost forgot. There's even more to update in the README.md. For instance, [the section on the controller.yaml](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/tree/master/pass_through_controllers#controller-yaml), and I guess [the examples no longer exist](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/tree/master/pass_through_controllers#examples) (they became integration tests)

the example controllers section got updated in #28

**Comment by fmauch** [comment]
Rebased so that #28 is included.

**Review by stefanscherzinger — APPROVED** [review]
:+1:

**Pull request merged**